core/server/services/auth/api-key/content.js

```diff
@@ -0,0 +1,46 @@
+const models = require('../../../models');
+const {
+    BadRequestError,
+    UnauthorizedError
+} = require('../../../lib/common/errors');
```

Comment:
I _think_ you should import `common` here to be consistent with rest of codebase and allow for search :relaxed:

Reply:
Oh yeah, I missed this one when moving everything back to `common.errors`